usng4j-api/src/main/java/org/codice/usng4j/UtmUpsCapabilities.java

```diff
@@ -24,12 +24,19 @@
   /** @return the northing value of this UTM coordinate. */
   double getNorthing();
 
+  /** @return the northing value with the NORTHING_OFFSET subtracted if NSIndicator equals SOUTH */
+  double getNorthingWithOffset();
+
   /** @return the zone number of this UTM coordinate. */
   int getZoneNumber();
 
   /** @return the latitude band for this UTM coordinate or null if not specified. */
+  @Deprecated
```

Comment:
✏️ Nit: should include a link to the new, correctly spelled `{@link #getLatitudeBand}` method.